Sources/Widgets/Widgets3D/AngleWidget/behavior.js

```diff
@@ -67,17 +68,18 @@ export default function widgetBehavior(publicAPI, model) {
   // --------------------------------------------------------------------------
 
   publicAPI.handleMouseMove = (callData) => {
+    const manipulator = model.activeState?.getManipulator
+      ? model.activeState.getManipulator()
+      : model.manipulator;
```

Comment:
This can be rewritten as `const manipulator = model.activeState?.getManipulator?.() ?? model.manipulator` for conciseness. Same with all of the other instances of this.